app/helpers/host_helper/textual_summary.rb

```diff
@@ -68,16 +70,20 @@ def textual_group_authentications
   end
 
   def textual_group_cloud_services
-    TextualGroup.new(_("Cloud Services"), textual_openstack_nova_scheduler) if @record.openstack_host?
+    return nil unless @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)
```

Comment:
Driveby review:

Seems like this conditional should be a "supported feature" instead hard coding this check in like this.

That said, probably doesn't need to happen now, just a code smell that I am noticing.

Reply:
Oh definitely agreed. I think that actually falls under https://github.com/ManageIQ/manageiq/issues/19440 :+1:

But that's a separate tasks, I'm not sure if there are features for this, or if this even still exists as a supported feature.